lib/tasks/instrumentation_generator/templates/newrelic.yml.tt

```diff
@@ -0,0 +1,19 @@
+---
+development:
+  error_collector:
+    enabled: true
+  apdex_t: 0.5
+  monitor_mode: true
+  license_key: bootstrap_newrelic_admin_license_key_000
+  instrumentation:
+    <%= @name.downcase %>: <%= $instrumentation_method %>
```

Comment:
I'm curious about this line. Does thor know to put something in the name spot, but also knows to skip the instrumentation_method part?

Reply:
Great catch! This isn't working as expected. I'll fix it 😄

Reply:
Resolved by: c75c8010404817ca14b55798000a97a889efc7dd